include/SFML/Graphics/Glsl.inl

```diff
@@ -106,7 +106,10 @@ struct Vector4
     ////////////////////////////////////////////////////////////
     /// \brief Construct from 4 vector components
     ///
-    /// \param X,Y,Z,W Components of the 4D vector
+    /// \param X Component of the 4D vector
+    /// \param Y Component of the 4D vector
+    /// \param Z Component of the 4D vector
+    /// \param W Component of the 4D vector
```

Comment:
As a side note, I'm personally a big fan of keeping things concise, including documentation. It's amazing how people write stuff like
```java
/**
 * Checks if the object is stored.
 * 
 * @param object Object that is checked.
 * @return true if the object is stored, false otherwise.
 */
boolean containsObject(Object object) { ... }
``` 
where the method name alone would contain 100% of the documentation's information, just to fit in some questionable document-everything-strictly policy. The only thing it does is becoming outdated at the next refactoring. But that as a side note :wink: